selenium/sample-build.js

```diff
@@ -32,61 +39,60 @@
-  const driver = new webdriver.Builder()
-    .usingServer(
+  const builder = new webdriver.Builder();
+  if (username) {
+    builder.usingServer(
       `https://${username}:${accessKey}@hub-cloud.browserstack.com/wd/hub`
-    )
-    .withCapabilities({
-      ...capabilities,
-      // Enable if using local environment
-      // 'browserstack.local': 'true',
-      ...(capabilities["browser"] && { browserName: capabilities["browser"] }), // Because NodeJS language binding requires browserName to be defined
-    })
-    .build();
+    );
+  } else {
+    builder.setChromeOptions(
+      new chrome.Options().addArguments("load-extension=browsers/dist")
+    );
+  }
+  builder.withCapabilities({
+    ...capabilities,
+    // Enable if using local environment
+    // 'browserstack.local': 'true',
+    ...(capabilities["browser"] && { browserName: capabilities["browser"] }), // Because NodeJS language binding requires browserName to be defined
+  });
+
+  const driver = await builder.build();
   await driver.get("https://www.pixiebrix.com");
   try {
     await driver.wait(webdriver.until.titleMatches(/pixiebrix/i), 5000);
     console.log(await driver.getTitle());
-    await driver.executeScript(
-      'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"passed","reason": "Title contains PixieBrix!"}}'
-    );
+    await driver.executeScript('console.log("sdasd")');
   } catch {
-    await driver.executeScript(
-      'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"failed","reason": "Page could not load in time"}}'
-    );
+    await driver.executeScript('console.log("error")');
   }
   await driver.quit();
 }
 
-const capabilities1 = {
-  browser: "chrome",
-  browser_version: "latest",
-  "bstack:options": {
-    os: "Windows",
-    osVersion: "10",
-  },
-  name: "Chrome test",
-};
-const capabilities2 = {
-  browser: "firefox",
-  browser_version: "latest",
-  "bstack:options": {
-    os: "Windows",
-    osVersion: "10",
-  },
-  name: "Firefox test",
-};
+async function getExtensionAsCrxInBase64() {
+  let buff = new Buffer.from(
+    fs.readFileSync("web-ext-artifacts/pixiebrix-1.1.12.zip")
+  );
+  return buff.toString("base64");
+}
+
+async function init() {
+  const capabilities1 = {
+    browser: "chrome",
+    browser_version: "latest",
+    "bstack:options": {
+      os: "Windows",
+      osVersion: "10",
+    },
+    name: "Chrome test",
+  };
 
-const capabilities3 = {
-  browser: "safari",
-  browser_version: "latest",
-  "bstack:options": {
-    os: "OS X",
-    osVersion: "Big Sur",
-  },
-  name: "Safari test",
-};
+  if (username) {
+    capabilities1.chromeOptions = {
+      extensions: [await getExtensionAsCrxInBase64()],
+    };
+  }
+
+  void testPixieBrixHomepage(capabilities1);
+}
 
-void testPixieBrixHomepage(capabilities1);
-void testPixieBrixHomepage(capabilities2);
-void testPixieBrixHomepage(capabilities3);
+init();
```

Comment:
Codacy has a fix for the issue: [Promises must be handled appropriately or explicitly marked as ignored with the `void` operator.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7734019)
```suggestion
void init();
```